javascript/stimulus_reflex.js

```diff
@@ -107,6 +100,7 @@ const extendStimulusController = controller => {
       element.reflexData = data
 
       dispatchLifecycleEvent('before', element)
+      adaptedConsumer.send(JSON.stringify({ channel }, data, stimulateOptions))
```

Comment:
With an options we have increased flexibility in how sending stuff over the wire works. This would allow a user to create an adapter that actually sends stuff over http instead of websockets. 

So the options object could contains something like `{"method": "POST"}` or similar that would be needed to send things over http.

Reply:
I think you got the parens/braces wrong here... you'll want to send the channel identifier as it is, and the data as stringified JSON:

```suggestion
      this.StimulusReflex.consumer.send(
        channel,
        JSON.stringify(data),
        stimulateOptions
      )
```

Reply:
You're right, I got that wrong here. I do think that the `channel` needs to be stringified as well. I'll have to test it to see what works. Once I've tested it, it'd be good if someone tested on rails too, to make sure.